modules/git/repo_archive.go

```diff
@@ -41,6 +45,8 @@ func ToArchiveType(s string) ArchiveType {
 	switch s {
 	case "zip":
 		return ZIP
+	case "tar":
+		return TAR
```

Comment:
Is this ever used? `RepoArchive` doesn't cover this case.

```suggestion
```

Reply:
Ohhh I see, you call this *from* the annex path. Nevermind then!

Reply:
Exactly, this seemed to be the cleanest option compared to using targz and immediately gunzip'ing or re-implementing the `git archive` call.